# Represents a maliput::api::LaneId.
string id
